sys-libs/libcap/metadata.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="project">
-	<email>base-system@gentoo.org</email>
-	<name>Gentoo Base System</name>
-</maintainer>
-<upstream>
-	<remote-id type="cpe">cpe:/a:libcap:libcap</remote-id>
-</upstream>
+	<maintainer type="project">
+		<email>base-system@gentoo.org</email>
+		<name>Gentoo Base System</name>
+	</maintainer>
+	<use>
+		<flag name="tools">Install captree, which requires dev-lang/go.</flag>
```

Comment:
Don't we *usually* avoid `.` in these? Also missing `<pkg></pkg>`

That aside, looks go to me.